packages/utils/src/lib/throttle.ts

```diff
@@ -44,19 +48,25 @@ function tick() {
 
 let started = false
 
+interface PseudoThrottleFn {
```

Comment:
maybe name this `CancellableFn` or `ThrottledFn` or something?

Reply:
actually, this had a hard time exporting for some reason - gonna extract this out for now.

Reply:
(exporting for the api docs, i mean)